lib/eventboss/launcher.rb

```diff
@@ -23,6 +23,8 @@ def initialize(queues, client, options = {})
 
     def start
       logger.info("Starting #{@workers.size} workers, #{@pollers.size} pollers", 'launcher')
+      setup_signals
```

Comment:
I believe it makes more sense to have the signal handling where the runner starts https://github.com/AirHelp/eventboss/blob/master/lib/eventboss/runner.rb#L17

I haven't checked if it will work well, but seems to be the best place for such logic